v8_c_api/src/v8_c_api.cpp

```diff
@@ -436,10 +436,16 @@ void v8_FreeContext(v8_context* ctx) {
 	V8_FREE(ctx);
 }
 
-void v8_SetPrivateData(v8_context* ctx, size_t index, void *pd) {
+int v8_SetPrivateData(v8_context* ctx, size_t index, void *pd) {
+	if (!pd) {
```

Comment:
Wonder if we should just assert it, do we really have an attention to handle such an error? I believe it is just something that should never happened right?